zerver/migrations/0372_invalid_characters_in_stream_names.py

```diff
@@ -0,0 +1,65 @@
+import unicodedata
+
+from django.db import connection, migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def fix_stream_names(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Stream = apps.get_model("zerver", "Stream")
+    Realm = apps.get_model("zerver", "Realm")
+
+    total_fixed_count = 0
+    realm_ids = Realm.objects.values_list("id", flat=True)
+    if not realm_ids:
```

Comment:
We generally prefer `len` checks for lists; it's more type-specific.  (E.g. `None` or 0 is falsey, but would be a bug if that is what `realm_ids` was)